### PR TITLE
supply some text for SCP-005

### LINKED ISSUE #17: SCP-005 is missing
This proposal was discussed in the first meeting, and accepted, but no proposal exists in the repository.

### PR DESCRIPTION
**Pull request by SethTisue**
fixes #17

### CODE REVIEW

File: proposals/005-continuity-of-scala-js.md [hunk 1]
```diff
@@ -1,6 +1,30 @@
...
+Proposed by Alexy Khabrov from Nitro
+
+## Proposal
+
+The proposal is for the Scala Center to ensure continuity of
+the Scala.js project, resources permitting.
+
+Background: At present (May 2016), the core Scala.js project members
+are Sébastien Doereane and Nicolas Stucki.  The funding for Stucki's
+position runs out in August.
```
Comment (julienrf) [inline proposals/005-continuity-of-scala-js.md]: Well, @nicolasstucki works more on Dotty than on Scala.js. The funding for @sjrd ends in August 2018 IIRC.

Comment (sjrd) [reply proposals/005-continuity-of-scala-js.md]: Note that this background was at the time of the meeting, i.e., May 2016. For that date, the background as stated is correct: Nicolas' funding on Scala.js ended in August 2016. He then started a Ph.D. of his own, on Dotty, in September 2016.

Re my funding, it basically ends when I graduate, which will likely be around April-June 2018.

**Comment by heathermiller** [comment]
This LGTM @SethTisue! Feel free to merge :)

**Comment by heathermiller** [comment]
(PS, thanks for updating this; it's been a total blindspot to us. I really appreciate it.)

**Pull request merged**